sdk/storage/azure-storage-internal-avro/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 12.1.0-beta.2 (Unreleased)
-
+## 12.1.0 (2021-09-15)
+### Dependency Updates
```

Comment:
Replace this with:
### Other Changes
#### Dependency Updates
- Updated `azure-storage-common` to version `12.13.0`